code/src/components/Game.js

```diff
@@ -0,0 +1,112 @@
+  const username = useSelector(store => store.rooms.username)
+  const room = useSelector(store => store.rooms)
+  // We'll use this variable to check if things are loading or not.
+  const isLoading = useSelector(store => store.loader.isLoading)
+  const dispatch = useDispatch();
+
+  const START_URL = "https://wk16-backend.herokuapp.com/start";
+  const ACTION_URL = "https://wk16-backend.herokuapp.com/action";
+
+  const startGameFetchInfo = {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({
+      username: username
+    })
+  }
+
+  // This function starts the game, by doing the first fetch from the API.
+  const startGame = () => {
+    dispatch(loader.actions.setLoading(true))
+    // Fetcha, och sedan skicka in responsen (data) till setGameState. Typ.
+    fetch(START_URL, startGameFetchInfo)
+      .then(response => response.json())
+      .then(data => {
+        // Pass "data" (the response from the API containing the coordinates, direction, etc) to the Game State.
+        dispatch(rooms.actions.setGameState(data))
+        dispatch(loader.actions.setLoading(false))
+      })
+  }
+
+  // This function is called on each click of a "Direction" button – i.e. the buttons in each action box with North, East, West or South.
+  const continueGame = (direction) => {
+    dispatch(loader.actions.setLoading(true))
+    fetch(ACTION_URL, {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({
+        username: username,
+        type: "move",
+        direction: direction
+      })
+    })
+      .then(response => response.json())
+      .then(data => {
+        dispatch(rooms.actions.setGameState(data))
+        dispatch(loader.actions.setLoading(false))
+      })
+  }
+
+  // This will make it run only the first time the component mounts. 
+  // "You can use the useEffect hook to run a function when the component has been mounted. By giving it an empty array as second argument it will only be run after the initial render."
+  useEffect(() => {
+    startGame();
+  }, []);
+
+  return (
+    <>
+      <div className="box-description">
+        <p>{room.gameState.coordinates}</p>
+        <p className="spacer-description">◆</p>
+        <p>"{room.gameState.description}"</p>
+      </div>
+
+      {/* Logic to display either one arrow, two arrows, or a "Game completed" text */}
+      {
+        (room.gameState.actions.length === 1)
+          ? <img className="image-arrow" src={arrow_one} alt="arrow" />
+          : null
+      }
+      {
+        (room.gameState.actions.length === 2)
+          ? <img className="image-arrow" src={arrow_split} alt="arrow" />
+          : null
+      }
+      {
+        (room.gameState.actions.length === 0)
+          ? <><h2 className="gameover-text">You completed the labyrinth!</h2><button onClick={() => window.location.reload()}>&gt; Restart game</button></>
+          : null
+      }
```

Comment:
Cool to see how you're writing the if else statements using the ternary operators. As an indicator for which arrow to show and what to show when the game is over! 👌